Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:3.10
```

Comment:
Is there is any substantial benefit? I had a look and it seemed the that it would result in a larger image in exchange for a slightly newer version of Ruby.

Reply:
It's just easier to handle as you don't have to worry about the ruby package name/version/... The image just provides it.

But yeah, the resulting image will double in size :(